test/manual/amp-autocomplete/amp-autocomplete.amp.html

```diff
@@ -1,28 +1,36 @@
-  <script async src="https://cdn.ampproject.org/v0.js"></script>
-  <script async custom-element="amp-autocomplete" src="https://cdn.ampproject.org/v0/amp-autocomplete-0.1.js"></script>
-  <script async custom-template="amp-form" src="https://cdn.ampproject.org/v0/amp-form-latest.js"></script>
-</head>
-<body>
-  <div height="1000">
-    <button id="focusButton" on="tap:input.focus">Focus</button>
-    <form method="post" action-xhr="/form/echo-json/post" target="_blank"
-      on="submit-success:AMP.setState({result: event.response})">
-      <amp-autocomplete id="autocomplete" filter="substring" min-characters="0">
-        <input type="text" id="input">
-        <script type="application/json" id="script">
-          { "items" : ["apple", "banana", "orange"] }
-        </script>
-      </amp-autocomplete>
-    </form>
-  </div>
-</body>
+  <head>
+    <meta charset="utf-8">
+    <title>amp-autocomplete examples</title>
+    <link rel="canonical" href="amps.html" >
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-autocomplete" src="https://cdn.ampproject.org/v0/amp-autocomplete-0.1.js"></script>
+    <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-latest.js"></script>
```

Comment:
Can we toggle the `amp-autocomplete` experiment on this file?

Reply:
I was planning on doing that when I update the file with an `inline` example and corresponding e2e test. I think this file only got linted in this change.

Reply:
Sounds good!